docs/source/components/spaces/link.md

```diff
@@ -0,0 +1,85 @@
+---
+title: <SpacesLink />
+---
+
+## Example
+
+```jsx
+import React from 'react';
+import Spaces, {
+  SpacesLink
+} from '@availity/spaces';
+
+<Spaces
+  spaceIds={['73162546201441126239486200007187']}
+  payerIds={['PayerID']}
+  clientId="my-client-id"
+>
+  <SpacesLink spaceId="73162546201441126239486200007187" body appIcon />
+</Spaces>;
+```
+
+## Props
+
+### `spaceId?: string`
+
+Required if `payerId` is not provided. The payer spaces ID of the payer to render the image for. If no `spaceId` or `payerId` is provided, the first space in the `spaces` array is used. Note: This is only to be used when the Spaces provider should only ever contain a single space.
+
+### `space?: SanitizedSpace`
+You don't have to use this component wrapped in the `SpacesProvider`. You can directly pass your `Space` into this, however, if you are wanting to take advantage of the sso links you will additionally need to pass the `clientId` in.
+
+### `clientId?: string`
+Required if you are passing your own `Space` in.
+
+### `tag?: React.ComponentType | string;`
+Tag to overwrite the root component rendered.
+
+### `bodyTag?: React.ComponentType | string;`
+Tag to overwrite the body component that renders the title, description and date values. It defaults to `CardBody` or `div` depending on the value of the `linkStyle` prop.
+
+### `linkStyle?: string`
+When passed in, provides predefined styles for the component. Possible values are `card`, and `list`.
+
+### `icon?: boolean`
+When `true`, will render an `@availity/icon` next to the title if present on the space.
```

Comment:
🎁  This is going to get whack if people start passing in their own `queries` bc they may not get `icons.navigation` etc. back even though they want to display it. 

Going to update the comment to denote that it will be part of the object in the `Spaces`